src/Swift.Bindings/src/Emitter/StringCSharpEmitter/Handler/MethodHandler.cs

```diff
@@ -189,7 +189,13 @@ private void EmitWrapperMethod(IndentedTextWriter writer, MethodEnvironment env)
             var pInvokeName = NameProvider.GetPInvokeName(methodDecl);
             var staticKeyword = methodDecl.MethodType == MethodType.Static || parentDecl is ModuleDecl ? "static " : "";
```

Comment:
Should we introduce a function for the modifiers? I see `static` and `unsafe` for now, we will have `async` soon, and potentially more in the future.

Reply:
I think this can be handled as a follow up, so far the use is very local